kirill/audit_report.md

```diff
@@ -0,0 +1,1051 @@
+  bounds defined by
+  the protocol (i.e. `MIN_PRICE` / `MAX_PRICE`).
+* Quote/collateral cannot be added/moved to the bucket at the same block after bucket becomes bankrupt.
+* Pool contract does not explicitly prevent internal arithmetic overflows caused by prolonged periods of high interest
+  rates. In practice, a maximum interest rate of 400% may break certain pool functionality after several years. Pools
+  with already extreme accumulator values close to causing `uint256` overflow in internal arithmetic should not be used.
+
+## Severity classification
+
+| **Severity**           | **Impact: High** | **Impact: Medium** | **Impact: Low** |
+|------------------------|------------------|--------------------|-----------------|
+| **Likelihood: High**   | Critical         | High               | Medium          |
+| **Likelihood: Medium** | High             | Medium             | Low             |
+| **Likelihood: Low**    | Medium           | Low                | Low             |
+
+**Impact** - the economic, technical, reputation or other damage to the protocol implied from a successful exploit.
+
+**Likelihood** - the probability that a particular finding or vulnerability gets exploited.
+
+**Severity** - the overall criticality of the particular finding.
+
+## Scope summary
+
+Reviewed whitepaper - https://www.ajna.finance/pdf/Ajna_Protocol_Whitepaper_10-12-2023.pdf (MD5 hash:
+ddc29820b45cd9664ff9ace5dd7bae1a)
+
+Reviewed
+commit [v0.10.0-rc8](https://github.com/ajna-finance/ajna-core/releases/tag/v0.10.0-rc8) ([dc49848696be62e107689be3295e85994964890c](https://github.com/ajna-finance/ajna-core/tree/dc49848696be62e107689be3295e85994964890c))
+
+Reviewed fixes commit - [v0.10.0-rc9](https://github.com/ajna-finance/ajna-core/releases/tag/v0.10.0-rc9) ([2d6bbcb39a020a041d5243eadd8f4cb77e85c41b](https://github.com/ajna-finance/ajna-core/tree/2d6bbcb39a020a041d5243eadd8f4cb77e85c41b))
+
+Reviewed contracts:
+
+- `src/interfaces/**`
+- `src/base/**`
+- `src/libraries/**`
+- `src/ERC20Pool.sol`
+- `src/ERC20PoolFactory.sol`
+- `src/ERC721Pool.sol`
+- `src/ERC721PoolFactory.sol`
+- `src/PoolInfoUtils.sol`
+- `src/PoolInfoUtilsMulticall.sol`
+- `src/PositionManager.sol`
+
+---
+
+# Findings Summary
+
+| ID     | Title                                                                                   | Severity      | Status       |
+|--------|-----------------------------------------------------------------------------------------|---------------|--------------|
+| [H-01] | Large borrowers bear a substantial economic risk of spurious liquidations               | High          | Acknowledged |
+| [M-01] | Reserve auction starting price calculation contradicts the whitepaper                   | Medium        | Fixed        |                                                                                                                                                                               
```

Comment:
H-02 also missing here.

Reply:
Wasn't included in his report here -> https://github.com/k1rill-fedoseev/audits/blob/master/solo/Ajna.md